From 74b4a34abfbab5fc1a3310951eaca9498409155a Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 4 Aug 2022 16:56:13 +0200
Subject: [PATCH] mpc85xx: p2020 enable: driver CONFIG_AT803X_PHY

This driver is for QCA8337N-AL3C chip, which is used as switch on Turris
1.x routers

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 target/linux/mpc85xx/p2020/config-default | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/linux/mpc85xx/p2020/config-default b/target/linux/mpc85xx/p2020/config-default
index ef32280dac..e6df5f4f1f 100644
--- a/target/linux/mpc85xx/p2020/config-default
+++ b/target/linux/mpc85xx/p2020/config-default
@@ -1,3 +1,4 @@
+CONFIG_AT803X_PHY=y
 CONFIG_CPU_RMAP=y
 CONFIG_GENERIC_CLOCKEVENTS_BROADCAST=y
 CONFIG_GENERIC_IRQ_MIGRATION=y
-- 
2.34.1

